specification/storagesync/resource-manager/Microsoft.StorageSync/preview/2020-03-01/examples/StorageSyncServices_Get.json

```diff
@@ -1,28 +1,41 @@
 {
-  "parameters": {
-    "subscriptionId": "52b8da2f-61e0-4a1f-8dde-336911f367fb",
-    "resourceGroupName": "SampleResourceGroup_1",
-    "storageSyncServiceName": "SampleStorageSyncService_1",
-    "syncGroupName": "SampleSyncGroup_1",
-    "api-version": "2020-03-01"
-  },
-  "responses": {
-    "200": {
-      "headers": {
-        "x-ms-request-id": "74e55a4d-1c6f-46de-9a8d-278e53a47403",
-        "x-ms-correlation-request-id": "d166ca76-dad2-49df-b409-d2acfd42d730"
-      },
-      "body": {
-        "name": "SampleStorageSyncService_1",
-        "location": "WestUS",
-        "tags": {},
-        "properties": {
-          "storageSyncServiceStatus": 0,
-          "storageSyncServiceUid": "\"2de01144-72da-4d7f-9d0c-e858855114a8\""
-        },
-        "id": "/subscriptions/3a048283-338f-4002-a9dd-a50fdadcb392/resourceGroups/SampleResourceGroup_1/providers/Microsoft.StorageSync/storageSyncServices/SampleStorageSyncService_1",
-        "type": "Microsoft.StorageSync/storageSyncServices"
-      }
+    "parameters": {
+        "subscriptionId": "52b8da2f-61e0-4a1f-8dde-336911f367fb",
+        "resourceGroupName": "SampleResourceGroup_1",
+        "storageSyncServiceName": "SampleStorageSyncService_1",
+        "syncGroupName": "SampleSyncGroup_1",
+        "api-version": "2020-03-01"
+    },
+    "responses": {
+        "200": {
+            "headers": {
+                "x-ms-request-id": "74e55a4d-1c6f-46de-9a8d-278e53a47403",
+                "x-ms-correlation-request-id": "d166ca76-dad2-49df-b409-d2acfd42d730"
+            },
+            "body": {
+                "name": "SampleStorageSyncService_1",
+                "location": "WestUS",
+                "tags": {},
+                "properties": {
+                    "storageSyncServiceStatus": 0,
+                    "storageSyncServiceUid": "\"2de01144-72da-4d7f-9d0c-e858855114a8\"",
+                    "privateEndpointConnections": [{
+                        "id": "subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.StorageSync/{resource}/{resourceName}/privateEndpointConnections/SampleStorageSyncService_1.cd99f12ba6f3483f9292229e4f822258",
+                        "properties": {
+                            "privateEndpoint": {
+                                "id": "subscriptions/52b8da2f-61e0-4a1f-8dde-336911f367fb/resourceGroups/SampleResourceGroup_2/providers/Microsoft.Network/privateEndpoints/testpe01"
+                            },
```

Comment:
Is privateEndpointConnections part of every GetStorageSyncService GET operaration now? Is this want we want/need? This will negatively impact performance. I'm wondering why are they not their own "resource".